package.json

```diff
@@ -8,7 +8,8 @@
     "build": "./script/build.sh",
     "build-analyze": "NODE_ENV=production npm run build -- --buildtype vagovprod --analyzer; npm run analyze",
     "build:compare": "node script/content-build-compare.js && diff --color websiteContentBuildHash.txt standaloneContentBuildHash.txt",
-    "build:content": "node --max-old-space-size=10240 script/build-content.js",
+    "build:content": "UNPUBLISHED_CONTENT=true node --max-old-space-size=10240 script/build-content.js",
```

Comment:
I will not merge this but need to test unpublished content on on review instance

Reply:
Looks like we're not using the  `UNPUBLISHED_CONTENT` var, so we should probably take it out.